app/src/main/java/com/kickstarter/ui/viewholders/ProjectViewHolder.java

```diff
@@ -169,7 +174,12 @@ public ProjectViewHolder(final @NonNull View view, final @NonNull Delegate deleg
     this.viewModel.outputs.blurbTextViewText()
       .compose(bindToLifecycle())
       .compose(observeForUI())
-      .subscribe(blurb -> this.blurbTextView.setText(Html.fromHtml(TextUtils.htmlEncode(blurb))));
+      .subscribe(this::setBlurbTextViews);
```

Comment:
What is the `blurbVariantTextView` for if we're only experimenting on the button?

Reply:
We are but there are 2 blurb containers. `blurb_view_variant` with the normal blurb and read more arrow (the whole container is clickable) and `blurb_view_variant` where **only** the button is clickable